test/integration/scenarios/network/peers_blacklist.js

```diff
@@ -151,21 +156,27 @@ module.exports = params => {
 						`${__dirname}/../../configs/config.node-0.json`,
 						JSON.stringify(params.configurations[0], null, 4)
 					);
-					restartNode('node_0');
+					// Restart the node to load the just changed configuration
+					common.restartNode('node_0');
```

Comment:
Please add the comment in other places also.

Reply:
I've done it.

Reply:
Where? I don't see any new changes since my last review.